arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesViewController.swift

```diff
@@ -56,28 +56,27 @@ class CreateSymbolStylesFromWebStylesViewController: UIViewController, UIAdaptiv
     
     // MARK: Methods
     
-    /// Create a map with an `AGSFeatureLayer` added to its operational layers.
-    /// - Parameter featureLayer: An `AGSFeatureLayer` object.
-    /// - Returns: An `AGSMap` object.
-    func makeMap(featureLayer: AGSFeatureLayer) -> AGSMap {
+    /// Create a map with a reference scale.
+    func makeMap(referenceScale: Double) -> AGSMap {
         let map = AGSMap(basemapStyle: .arcGISLightGray)
-        map.referenceScale = 1e5
-        map.operationalLayers.add(featureLayer)
+        map.referenceScale = referenceScale
```

Comment:
I realized that if I add the feature layer before setting its unique value renderer, the layer may load with its default renderer/symbols first (displays as black dots). So instead I add the feature layer to the map *after* setting its renderer.

This afternoon I'll discuss with Divesh to learn a bit more about how feature layer and its renderer works.